Emby.Naming/TV/SeasonPathParser.cs

```diff
@@ -86,7 +86,7 @@ private static (int? SeasonNumber, bool IsSeasonFolder) GetSeasonNumberFromPath(
                 }
             }
 
-            if (filename.StartsWith('s'))
+            if (filename.Length > 0 && char.ToLowerInvariant(filename[0]) == 's')
```

Comment:
```suggestion
            if (filename.Length > 0 && (filename[0] == 'S' || filename[0] == 's'))
```

Since we know the only two options this is likely better